src/main.rs

```diff
@@ -5,11 +5,22 @@ mod memory;
 mod seq;
 mod state;
 
-use crate::config::load_config;
+use crate::config::{load_config, Config};
 use crate::gui::Gui;
+use log::*;
 
-fn main() {
+fn main() -> Result<(), Box<dyn std::error::Error>> {
     colog::init();
-    let conf = load_config("config.yaml").ok();
+
+    // Loads the config.toml if it exists or loads defaults
+    let conf = match load_config("./config.toml") {
+        Ok(conf) => conf,
+        Err(_err) => {
+            warn!("No config.toml found; using defaults.");
```

Comment:
Technically, no config **loaded**; file not found or an error when parsing it. There's an internal `error!` on the latter so this is fine.